Networking/Networking/Model/WordPressComUser.swift

```diff
@@ -0,0 +1,65 @@
+import Codegen
+import Foundation
+
+/// Basic information of a WordPress.com user
+public struct WordPressComUser: Decodable, GeneratedFakeable, GeneratedCopiable {
```

Comment:
nit: since we already have `DotcomDevice`, does it make sense to keep the naming consistent?
```suggestion
public struct DotcomUser: Decodable, GeneratedFakeable, GeneratedCopiable {
```